### ISSUE TITLE
Starting simulation managers on Zeus cluster with on-site monitoring does not work

### ISSUE THREAD

**Issue by kliput** [Scalarm/scalarm_experiment_manager#176]
Using username/password credentials set for Zeus Cluster.

Trying to start one simulation manager with on-site monitoring. The simulation manager is stuck in created state.

If on-site monitoring is disabled, it works.

**Comment by kliput** [comment]
Schedule simulation managers request has parameter: ``infrastructure_name:cluster_5767ac524269a824606d7083``

**Comment by kliput** [comment]
It seems that Zeus couldn't be reached:
```
2017-03-02 08:06:37 [cluster_5767ac524269a824606d7083][Exp: 58b7c5fe4cc04e01f3896115][SiM: 58b7d2614cc04e01f38963aa][Res: ] - State: created, Resource status: not_available
```
Checking the cause...

**Comment by kliput** [comment]
The credentials object is valid (checked with ``ClusterCredentials#ssh_session`` method).

**Comment by kliput** [comment]
Onsite monitoring log on Zeus:
```
2017/03/05 18:35:51 [I] Checking for working monitoring mark
2017/03/05 18:35:51 [I] Mark file exists...
2017/03/05 18:35:51 [I] ...but no process with saved pid [30417] is working
2017/03/05 18:35:51 [I] Creating monitoring mark file, pid: 800
2017/03/05 18:35:51 [I] Config loaded
2017/03/05 18:35:51 [I]   Information Service address: system.scalarm.com/information
2017/03/05 18:35:51 [I]   Login:                       <some_login>
2017/03/05 18:35:51 [I]   Scalarm certificate path:    
2017/03/05 18:35:51 [I]   Scalarm scheme:              https
2017/03/05 18:35:51 [I]   Insecure SSL:                true
2017/03/05 18:35:51 [I]   Exit timeout (secs):         0
2017/03/05 18:35:51 [I]   Probe frequency (secs):      0
2017/03/05 18:35:51 [I]   Verbose mode:                false
2017/03/05 18:35:51 [I]   Infrastructures:             [{cluster_5767ac524269a824606d7083.pbs }]
2017/03/05 18:35:51 [I] Configuration finished
2017/03/05 18:35:51 [I] RepetitiveCaller : call GetSimulationManagerRecords failed, err: 
Damaged data
Reattempt in 15 s
2017/03/05 18:36:06 [I] RepetitiveCaller : call GetSimulationManagerRecords failed, err: 
Damaged data
Reattempt in 15 s
2017/03/05 18:36:21 [I] RepetitiveCaller : call GetSimulationManagerRecords failed, err: 
Damaged data
Reattempt in 15 s
2017/03/05 18:36:36 [I] RepetitiveCaller : call GetSimulationManagerRecords failed, err: 
Damaged data
Reattempt in 15 s
2017/03/05 18:36:51 [I] RepetitiveCaller : call GetSimulationManagerRecords failed, err: 
Damaged data
Reattempt in 15 s
```

This error is connected to: https://github.com/Scalarm/scalarm_experiment_manager/issues/177

**Comment by kliput** [comment]
Working on fix on branch:
https://github.com/Scalarm/scalarm_experiment_manager/tree/issue/176-onsite-monitoring-clusters

**Comment by kliput** [comment]
#178 should fix wrong infrastructure name in onsite monitoring, but onsite still does not work:

```
2017/03/05 21:33:00 [I] ScalarmWorkersManager 2016.04.08-1
2017/03/05 21:33:00 [I] Checking for working monitoring mark
2017/03/05 21:33:00 [I] Mark file exists...
2017/03/05 21:33:00 [I] ...but no process with saved pid [21479] is working
2017/03/05 21:33:00 [I] Creating monitoring mark file, pid: 9788
2017/03/05 21:33:00 [I] Config loaded
2017/03/05 21:33:00 [I]   Information Service address: system.scalarm.com/information
2017/03/05 21:33:00 [I]   Login:                       <some_login>
2017/03/05 21:33:00 [I]   Scalarm certificate path:    
2017/03/05 21:33:00 [I]   Scalarm scheme:              https
2017/03/05 21:33:00 [I]   Insecure SSL:                true
2017/03/05 21:33:00 [I]   Exit timeout (secs):         0
2017/03/05 21:33:00 [I]   Probe frequency (secs):      0
2017/03/05 21:33:00 [I]   Verbose mode:                false
2017/03/05 21:33:00 [I]   Infrastructures:             [{cluster_5767ac524269a824606d7083 }]
2017/03/05 21:33:00 [I] Configuration finished
2017/03/05 21:33:00 [I] [cluster_5767ac524269a824606d7083] 3 sm_records
2017/03/05 21:33:00 [I] Deleting monitoring mark file
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x40 pc=0x40c367]

goroutine 1 [running]:
main.main()
	/home/scalarm/repo_build/src/github.com/Scalarm/scalarm_workers_manager/scalarmWorkersManager.go:148 +0x1fd7

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 17 [select, locked to thread]:
runtime.gopark(0x8379e8, 0xc82002c728, 0x7a3a20, 0x6, 0x43f818, 0x2)
	/usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc82002c728, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc82002c728)
	/usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:227 +0x353
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 7 [chan receive]:
main.SignalCatcher(0xc82001c300, 0xc82001c360, 0x7ffc35c723d7, 0x16)
	/home/scalarm/repo_build/src/github.com/Scalarm/scalarm_workers_manager/signalHandler.go:16 +0x152
created by main.main
	/home/scalarm/repo_build/src/github.com/Scalarm/scalarm_workers_manager/scalarmWorkersManager.go:46 +0x25c

goroutine 19 [IO wait]:
net.runtime_pollWait(0x7f94e277eb78, 0x72, 0xc8200101d0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8203100d0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203100d0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820310070, 0xc820353000, 0x1000, 0x1000, 0x0, 0x7f94e2775050, 0xc8200101d0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820314008, 0xc820353000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
crypto/tls.(*block).readFromUntil(0xc8200b86c0, 0x7f94e2735280, 0xc820314008, 0x5, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:455 +0xcc
crypto/tls.(*Conn).readRecord(0xc820332000, 0x837a17, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:540 +0x2d1
crypto/tls.(*Conn).Read(0xc820332000, 0xc820335000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:901 +0x167
net/http.noteEOFReader.Read(0x7f94e2735688, 0xc820332000, 0xc8200d8058, 0xc820335000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/transport.go:1370 +0x67
net/http.(*noteEOFReader).Read(0xc82030ff40, 0xc820335000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	<autogenerated>:126 +0xd0
bufio.(*Reader).fill(0xc8200be900)
	/usr/local/go/src/bufio/bufio.go:97 +0x1e9
bufio.(*Reader).Peek(0xc8200be900, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/bufio/bufio.go:132 +0xcc
net/http.(*persistConn).readLoop(0xc8200d8000)
	/usr/local/go/src/net/http/transport.go:876 +0xf7
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:685 +0xc78

goroutine 20 [select]:
net/http.(*persistConn).writeLoop(0xc8200d8000)
	/usr/local/go/src/net/http/transport.go:1009 +0x40c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:686 +0xc9d
```

**Comment by dkrol** [comment]
This issue has been fixed in docker image 1.0.2. Production @ system.scalarm.com has been updated.
